Curve-finance-suite/src/agents/cross.asset.swap.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from 'forta-agent';
+
+export const CROSS_CHAIN_SWAP_SIGNATURE =
+  'TokenUpdate(uint256,address,address,uint256)';
+
+const createFinding = (alertID: string, address: string): Finding => {
+  return Finding.fromObject({
+    name: 'CrossChainSwap funciton called',
+    description: 'CrossChainSwap funciton called on pool',
+    alertId: alertID,
+    severity: FindingSeverity.Low,
+    type: FindingType.Suspicious,
+    metadata: {
+      address: address,
+    },
+  });
+};
+
+export default function provideCrossAssetSwap(
+  alertID: string,
+  address: string
+): HandleTransaction {
+  return async (txEvent: TransactionEvent): Promise<Finding[]> => {
+    const findings: Finding[] = [];
+
+    if (txEvent.addresses[address] == false) return findings;
```

Comment:
Same as the previous suggestions
```suggestion
    if (!txEvent.addresses[address]) return findings;
```